static/styles/subscriptions.css

```diff
@@ -680,8 +680,8 @@ h4.user_group_setting_subsection_title {
 #subscription_overlay {
     #user-group-creation,
     #stream-creation {
-        max-height: calc(100% - 102px);
-        overflow: auto;
+        max-height: calc(100% - 60px);
+        overflow: hidden;
```

Comment:
This change also affects other elements; is that intended?

Why is the `overflow` property changed? I tried with `overflow: auto` and didn't see the native scrollbar as mentioned in this [comment](https://github.com/zulip/zulip/pull/21313#discussion_r820083023).

Reply:
No, this is not intended, i separated this into a own setting.

I falsely kept the change to the overflow property as i started with the other pr and build on it, but i removed this change.